lib/Debug/DebugInfo.cpp

```diff
@@ -0,0 +1,34 @@
+/// \file DebugInfo.cpp
+/// \brief Run an instance of DebugHelper over the IR.
+
+#include "llvm/Pass.h"
+#include "llvm/IR/AssemblyAnnotationWriter.h"
+#include "llvm/IR/IRBuilder.h"
+
+#include "revng/Support/CommandLine.h"
+#include "revng/Support/Debug.h"
+#include "revng/Support/DebugHelper.h"
+#include "revng/Support/revng.h"
+
+using namespace llvm;
+
+namespace {
+
+class DebugInfo : public ModulePass {
+public:
+    static char ID;
+    DebugInfo() : ModulePass(ID) { }
+
+    bool runOnModule(Module &M);
+};
+
+}
```

Comment:
Keeping this in a `.cpp` file forces us to use this pass from `opt` only. Please move this to a public header.